test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1019,7 +1019,7 @@ def run_cmd(self, hosts=None, cmd=None, sudo=False, stdin=None,
 
             if as_script:
                 # must pass as_script=False otherwise it will loop infinite
-                if platform == 'shasta' and not runas:
+                if platform == 'shasta' and runas:
```

Comment:
I think we can just pass 'runas' to rm() in either case, did you see any issue doing that in the generic case?